sdk/go/auto/workspace.go

```diff
@@ -61,6 +61,8 @@ type Workspace interface {
 	GetConfigWithOptions(context.Context, string, string, *ConfigOptions) (ConfigValue, error)
 	// GetAllConfig returns the config map for the specified stack name, scoped to the current workspace.
 	GetAllConfig(context.Context, string) (ConfigMap, error)
+	// GetAllConfig returns the config map for the specified stack name, scoped to the current workspace.
```

Comment:
Doc comment needs updating

Reply:
I ended up removing the Get and Set config functionality from this PR as it probably should be tracked separately